modules/react/layout/lib/utils/systemProps.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  CanvasBorderRadiusKeys,
+  CanvasColor,
+  CanvasDepth,
+  CanvasSpaceKeys,
+  CanvasTypeProperties,
+} from '@workday/canvas-kit-react/tokens';
+
+export type SystemPropNames =
+  | 'color'
+  | 'depth'
+  | 'font'
+  | 'fontSize'
+  | 'fontWeight'
+  | 'shape'
+  | 'space';
+
+/**
+ * Canvas System Prop Values
+ * ---
+ * These system prop values are restricted to Canvas token values.
+ */
+export type CanvasSystemPropValues = {
+  color: CanvasColor;
+  depth: keyof CanvasDepth;
+  font: keyof CanvasTypeProperties['fontFamilies'];
+  fontSize: keyof CanvasTypeProperties['fontSizes'];
+  fontWeight: keyof CanvasTypeProperties['fontWeights'];
+  shape: CanvasBorderRadiusKeys;
+  space: CanvasSpaceKeys;
+};
+
+/**
+ * System Prop Values
+ * ---
+ * These system prop values include Canvas System Prop Values and generic values.
+ */
+export type SystemPropValues = {
```

Comment:
This is another nice improvement. We're consolidating all system prop values to one location.